### ISSUE TITLE
[a11y] VideoEmbedInnerWeb move accessibilityLabel/Hint from generic View to <figure>

### ISSUE THREAD

**Issue by 0ddfell0w** [bluesky-social/social-app#9837]
Quick accessibility improvement suggestion for video embeds on web.

See https://github.com/bluesky-social/social-app/blob/18ab5d6f34f87c2851e02fc4c64afb07f6b6f475/src/components/Post/Embed/VideoEmbed/VideoEmbedInner/VideoEmbedInnerWeb.tsx#L57

Currently the accessibility label is applied to a generic div (<View>) and the <figure> element has no label. 
Proposal: move accessibilityLabel to the <figure> element. Alternatively, you could add role=group to the <View>
